Podfile.lock

```diff
@@ -58,9 +58,9 @@ SPEC CHECKSUMS:
   RxNuke: 7c56f40f26bc3ec715cff5f5355119ca7989b010
   RxRelay: 89d54507f4fd4d969e6ec1d4bd7f3673640b4640
   RxSwift: e2dc62b366a3adf6a0be44ba9f405efd4c94e0c4
-  TinyNetworking: f373ddd43edc5320ff98059ff484591970fbfb17
+  TinyNetworking: c0b2cbfe8db8451e42c3371a3eecfc61538b310a
```

Comment:
Was there a reason that you updated the TinyNetworking dependency?

Reply:
This is an update I didn't expect, after running `pod install` this line changed. Will reset this one.

Reply:
Sorry I installed another package for generating random names so this can't be reverted 😢

Reply:
That's fine - something look out for in the future. 

Generally, I try to stay on top of updates to dependencies, even without a reason for it, but I do like to sequester them into their own PR when I make the change, just so any breakage can be isolated.